domains/certificates/Query.API/API/MeasurementsSyncer/EnergyMeasuredIntegrationEventHandler.cs

```diff
@@ -53,10 +53,9 @@ private CertificateDto CreateCertificate(EnergyMeasuredIntegrationEvent measurem
 
         if (measurementEvent.MeterType == MeterType.Production)
         {
-            var address = measurementEvent.Address;
-            hashedAttributes.Add(new HashedAttribute
-            { Key = EnergyTagAttributeKeys.EnergyTagProductionDeviceUniqueIdentification, Value = measurementEvent.GSRN });
-            hashedAttributes.Add(new HashedAttribute { Key = EnergyTagAttributeKeys.EnergyTagProductionDeviceLocation, Value = address });
+            hashedAttributes.Add(new HashedAttribute { Key = EnergyTagAttributeKeys.EnergyTagProductionDeviceUniqueIdentification, Value = measurementEvent.GSRN });
+            hashedAttributes.Add(new HashedAttribute { Key = EnergyTagAttributeKeys.EnergyTagProductionDeviceLocation, Value = measurementEvent.Address.ToString() });
+            hashedAttributes.Add(new HashedAttribute { Key = EnergyTagAttributeKeys.EnergyTagProductionDeviceCapacity, Value = measurementEvent.Capacity });
```

Comment:
Hvis man ændrer integrationEvent, er det så et breaking change på consumer siden?

Reply:
Ja der kan potentielt godt ende beskeder i rabbit, der ikke kan consumes. Derfor jeg slukker measurementssyncer inden deploy (se description)

"It's a contract change, so I'll turn off measurements syncer before merging this PR."

Reply:
Okay